boot/makebootable.go

```diff
@@ -298,7 +301,13 @@ func MakeRecoverySystemBootable(rootdir string, relativeRecoverySystemDir string
 		"snapd_recovery_kernel": filepath.Join("/", kernelPath),
 	}
 	if _, ok := bl.(bootloader.TrustedAssetsBootloader); ok {
-		recoveryCmdlineArgs, err := bootVarsForTrustedCommandLineFromGadget(bootWith.GadgetSnapOrDir)
+		// Look at gadget default values for system.kernel.*cmdline-append options
+		cmdlineOpt, err := buildOptionalKernelCommandLine(bootWith.GadgetSnapOrDir, model)
+		if err != nil {
+			return fmt.Errorf("while retrieving optional kernel command line: %v", err)
```

Comment:
given the new naming is not clear that optional kernel command can be understood?

system.kernel.*cmdline-append defaults instead?